backend/app/api/admin_routes/retrieve.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional, List
+
+from fastapi import APIRouter
+from app.models import Document
+from app.api.deps import SessionDep, CurrentSuperuserDep
+from app.rag.retrieve import RetrieveService
+
+router = APIRouter()
+
+
+@router.get("/admin/retrieve/documents")
```

Comment:
Why put api like this under `/admin/` path? It's just for general usage rather than for administration, and can be used by even common user.

Reply:
This retrieve API is extremely heavy, it might cost above 60 MB json in one go. Besides that, it also exposure our raw data, so I think we might need to put it into the `/admin` field.

Reply:
Get, partly make sense.

Reply:
The 60 MB is an error. I retrieved too many documents. But it is also exposing our row data. It's only data security aspect thinking.

Reply:
Got it. Then we leave it here. It's not hard to change it if we decide.

Reply:
So big...